packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -147,7 +178,8 @@
 
   .iui-active::after {
     @media (prefers-reduced-motion: no-preference) {
-      transition: width var(--iui-duration-1) ease, height var(--iui-duration-1) ease;
+      transition: width var(--iui-duration-1) ease,
```

Comment:
unintentional formatting changes :(

Reply:
Yes, I noticed that when making this PR ready for review. But instead of reverting it, I thought of keeping it, similar to https://github.com/iTwin/iTwinUI/pull/935#discussion_r1057810187

Reply:
should be revertable now that #950 is merged

Reply:
Yes, it was reverted after merging with `main`